modules/ROOT/pages/_partials/task-add-api-instance.adoc

```diff
@@ -99,7 +99,9 @@ See xref:runtime-fabric::proxy-deploy-runtime-fabric.adoc[Deploying API Proxies
 [arabic]
 .. Click the API from the list under **Select API**. You can search for a specific API if needed.
 .. Update the *Asset type*, *API version*, and *Asset version* if needed.
-
+.. If you chose a *RAML/OAS* asset type, view the *Conformance Status* of the API to ensure the API is conformant. However, you must fix some conformance issues, such as applying policies, after deploying the API instance.
```

Comment:
I don't understand what you're trying to say here. Is the non-conformant label not accurate if they haven't yet deployed the API instance? Would some instances already be deployed so that the label is accurate? Also, chose is past tense, which is against our standard. Would the following be accurate?
For deployed instances that are *RAML/OAS* asset type, if *Conformance Status* is non-conformant, view the *Governance Report* to view information needed to find and fix the conformance issues. See ...

Reply:
I wouldn't put an example of what the conformance issue might be here, but leave that to the linked topic. Otherwise, there is unnecessary duplication and too much too soon.